erpnext/selling/doctype/quotation/quotation.js

```diff
@@ -206,7 +206,7 @@ cur_frm.cscript['Declare Order Lost'] = function(){
 
 }
 
-frappe.ui.form.on("Quotation Item", "items_on_form_rendered", function(frm, cdt, cdn) {
+frappe.ui.form.on("Quotation Item", "items_on_form_rendered", "packed_items_on_form_rendered", function(frm, cdt, cdn) {
```

Comment:
I don't think this works, please show an animated screenshot that this works.

Reply:
**Before had no serial button:**


![noserialbutton](https://user-images.githubusercontent.com/14797383/48021460-dc3cad80-e11f-11e8-8797-e85f98bdb8d9.JPG)


After coding:


![gif](https://user-images.githubusercontent.com/14797383/48021501-f080aa80-e11f-11e8-8ee2-da4d9b442811.gif)